examples/nextjs/src/components/layout/header.js

```diff
@@ -8,9 +11,21 @@ function Header({ children }) {
                     <Link href="/">TravelLux in NextJS</Link>
                 </h2>
             </div>
+            <ReorderButton />
             {children}
         </header>
     );
 }
 
+function ReorderButton() {
+
+    return (
+        <button className="bg-[#426BF0] rounded-sm flex cursor-pointer border-none px-2 py-1 gap-2"  onClick={() => reorderMenu()} >
+            <Image src="/assets/icons/arrow-up.svg" width="0" height="0" className='w-6 h-6' alt="arrow up" loader={PublicImageLoader}/>
+            <Image src="/assets/icons/arrow-down.svg"  width="0" height="0" className='w-6 h-6' alt="arrow down" loader={PublicImageLoader}/>
```

Comment:
Should we add assets to the example?


cc: @fmontes

Reply:
Added